src/main/java/seedu/address/logic/commands/FindStatusCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.StatusContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose status contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindStatusCommand extends Command {
+
+    public static final String COMMAND_WORD = "findstatus";
```

Comment:
Should the COMMAND_WORD be find_by_status to follow what we have in our user guide?